tools/jbrowse/jbrowse_to_container.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0"?>
+<tool id="jbrowse_to_container" name="Add organisms to JBrowse container" version="@WRAPPER_VERSION@">
+    <description>will replace any existing organism</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements"/>
+    <command detect_errors="aggressive"><![CDATA[
+
+        if [ -z "@DATA_DIR@" ]; then
+            echo '@DATA_DIR@ variable must be set to use this tool';
+            exit 1;
+        fi
+
+        &&
+
+        rm -rf "@DATA_DIR@/"*
```

Comment:
This wipes out *everything* in the shared data_dir? Shouldn't it be `@DATA_DIR@/${dataset_name}/`?